src/PlayTurn.java

```diff
@@ -1,75 +1,94 @@
+import java.io.*;
 import java.util.Scanner;
 
 public class PlayTurn {
-    //setting the object for the winCondition class 
-    WinCondition winCondition=new WinCondition();
-    MainGame mainGame=new MainGame();
-    
-  //a method to confirm the player symbol
-    public static void settingPlayerSymbol(int playerNo,char playerSymble)
-    {
+    // setting the object for the winCondition class
+    MainGame mainGame = new MainGame();
+
+   
+    /**
+     * a method to confirm the player symbol
+     * @param playerNo : the number of the player that is playing
+     * @param playerSymble : the Player Symbol
+     */
+    public static void settingPlayerSymbol(int playerNo, char playerSymble) {
         Scanner scan = new Scanner(System.in);
-        System.out.println("Player "+playerNo+" YOUR SYMBOL IS:"+playerSymble+"\nIs that Correct?");
-        char confirm=scan.next().charAt(0);
-        //Confirming player Symbol
-        if(confirm=='n'||confirm=='N')
-        {
-            System.out.println("Player "+playerNo+" ENTER YOUR NEW SYMBOL:");
+        System.out.println("Player " + playerNo + " YOUR SYMBOL IS:" + playerSymble + "\nIs that Correct?");
+        char confirm = scan.next().charAt(0);
+        // Confirming player Symbol
+        if (confirm == 'n' || confirm == 'N') {
+            System.out.println("Player " + playerNo + " ENTER YOUR NEW SYMBOL:");
             playerSymble = scan.next().charAt(0);
         }
     }
-    
-    
```

Comment:
remove spaces

Reply:
red mean the old file not the pull requested file there is no spaces in the code in this lines